lambdas/src/SmartContentServerFetcher.ts

```diff
@@ -1,28 +1,28 @@
 import log4js from "log4js"
-import fetch from "node-fetch"
+import { Fetcher, RequestOptions } from "dcl-catalyst-commons";
 
 /**
  * This fetcher tries to use the internal docker network to connect lambdas with the content server.
  * If it can't, then it will try to contact it externally
  */
-export class SmartContentServerFetcher {
+export class SmartContentServerFetcher extends Fetcher {
 
     private static INTERNAL_CONTENT_SERVER_URL: string = `http://content-server:6969`
     private static LOGGER = log4js.getLogger('SmartContentServerFetcher');
 
     private contentServerUrl: string | undefined
 
-    constructor(private readonly externalContentServerUrl: string) { }
+    constructor(private readonly externalContentServerUrl: string) {
+        super()
+    }
 
     async getContentServerUrl(): Promise<string> {
```

Comment:
Can we make this private now?

Reply:
No, because sometimes you want to know this externally (at leas to log the URL).